.github/workflows/submitty_ci.yml

```diff
@@ -273,333 +274,197 @@ jobs:
 
-          - name: Switch to LDAP auth
-            run:  sudo sed -ie "s/Database/Ldap/g" ${SUBMITTY_INSTALL_DIR}/config/authentication.json
+        - name: Setup SAML
+          run: |
+            cd $SUBMITTY_REPOSITORY
+            sudo -E env "PATH=$PATH" bash .setup/testing/setup_saml.sh
 
-          - name: Run cypress e2e login tests with LDAP auth
-            uses: cypress-io/github-action@v5
-            with:
-              config: baseUrl=http://localhost
-              spec: cypress/e2e/login.spec.js
-              working-directory: ${{env.SUBMITTY_REPOSITORY}}/site
-              browser: chrome
+        - name: Switch to SAML auth
+          run:  sudo sed -ie "s/Ldap/Saml/g" ${SUBMITTY_INSTALL_DIR}/config/authentication.json
+
+        - name: Run cypress e2e login tests with SAML auth
+          uses: cypress-io/github-action@v5
+          with:
+            config: baseUrl=http://localhost,chromeWebSecurity=false
+            spec: cypress/e2e/login.spec.js
+            working-directory: ${{env.SUBMITTY_REPOSITORY}}/site
+            browser: chrome
 
-          - name: Setup SAML
-            run: |
-              cd $SUBMITTY_REPOSITORY
-              sudo -E env "PATH=$PATH" bash .setup/testing/setup_saml.sh
+        - uses: actions/upload-artifact@v3
+          if: failure()
+          with:
+            name: cypress-screenshots
+            path: ${{env.SUBMITTY_REPOSITORY}}/site/cypress/screenshots
+        - uses: actions/upload-artifact@v3
+          if: failure()
+          with:
+            name: cypress-videos
+            path: ${{env.SUBMITTY_REPOSITORY}}/site/cypress/videos
 
-          - name: Switch to SAML auth
-            run:  sudo sed -ie "s/Ldap/Saml/g" ${SUBMITTY_INSTALL_DIR}/config/authentication.json
+        - name: Check for site errors
+          working-directory: ${{ env.SUBMITTY_REPOSITORY }}
+          run: sudo bash tests/test_site_error_log.sh
 
-          - name: Run cypress e2e login tests with SAML auth
-            uses: cypress-io/github-action@v5
-            with:
-              config: baseUrl=http://localhost,chromeWebSecurity=false
-              spec: cypress/e2e/login.spec.js
-              working-directory: ${{env.SUBMITTY_REPOSITORY}}/site
-              browser: chrome
 
-          - uses: actions/upload-artifact@v3
-            if: failure()
-            with:
-              name: cypress-screenshots
-              path: ${{env.SUBMITTY_REPOSITORY}}/site/cypress/screenshots
-          - uses: actions/upload-artifact@v3
-            if: failure()
-            with:
-              name: cypress-videos
-              path: ${{env.SUBMITTY_REPOSITORY}}/site/cypress/videos
 
-          - name: Check for site errors
-            working-directory: ${{ env.SUBMITTY_REPOSITORY }}
-            run: sudo bash tests/test_site_error_log.sh
+    e2eIntegrationTests:
+      runs-on: ubuntu-20.04
+      services:
+        postgres:
+          image: postgres
```

Comment:
It would be good to add a comment here stating why postgres is listed here, as well a TODO to remove it eventually.